styles.css

```diff
@@ -83,6 +83,17 @@
 /***************************************************/
 /* Put Selector post-it styles here! ↓ */
 
+ul{
```

Comment:
You should be styling the list element that has your name as an id. This will target the ul element that contains all of the Selector post its